examples/child_window.rs

```diff
@@ -3,7 +3,7 @@
 mod fill;
 
 #[cfg(any(x11_platform, macos_platform, windows_platform))]
-fn main() {
+fn main() -> std::result::Result<(), impl std::error::Error> {
```

Comment:
```suggestion
fn main() -> Result<(), impl std::error::Error> {
```
This seemed to work for me locally.

Reply:
yeah, I'll drop the fully qualified name